index.html

```diff
@@ -140,7 +140,35 @@ <h1 class="title_post">Multi-Post Stories</h1>
   
       <section class="button">See Project</section>
     </section>
-
+    <section id="contact-me">
+      <header>
+        <h2>CONTACT US</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
```

Comment:
- [ ] Kindly stick to the Figma design as much possible  to Make your project looks good. you can do so by enhancing colors ,font-sizes,font-weight,etc.

Figma design:
![image](https://user-images.githubusercontent.com/90524466/186189668-d4aff633-5781-4005-9ca6-b95f6abd6e3a.png)

Your design:
![image](https://user-images.githubusercontent.com/90524466/186189777-faaa8331-522a-4006-b0b4-84edc9e33013.png)